sys-fs/btrfsmaintenance/btrfsmaintenance-0.3.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils
+
+DESCRIPTION="Scripts for btrfs maintenance tasks like periodic scrub, balance, trim or defrag"
+HOMEPAGE="https://github.com/kdave/btrfsmaintenance"
+SRC_URI="https://github.com/kdave/btrfsmaintenance/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	sys-fs/btrfs-progs
+	virtual/cron"
+
+src_configure() {
+	sed -i -e 's#/etc/sysconfig/btrfsmaintenance#/etc/conf.d/btrfsmaintenance#' *.sh || die
```

Comment:
As you can see, upstream uses /etc/sysconfig (SuSE Linux equivalent to Gentoo /etc/conf.d). On my Gentoo systems /etc/sysconfig does not exist, and I dislike creating it now only for this package. So what about /etc ?